dipy/data/fetcher.py

```diff
@@ -666,3 +703,92 @@ def read_cenir_multib(bvals=None):
 
 fetch_cenir_multib.__doc__ += CENIR_notes
 read_cenir_multib.__doc__ += CENIR_notes
+
+
+def read_viz_icons(style='icomoon', fname='infinity.png'):
+    """ Read specific icon from specific style
+
+    Parameters
+    ----------
+    style : str
+        Current icon style. Default is icomoon.
+    fname : str
+        Filename of icon. This should be found in folder HOME/.dipy/style/.
+        Default is infinity.png.
+
+    Returns
+    --------
+    path : str
+        Complete path of icon.
+
+    """
+
+    folder = pjoin(dipy_home, 'icons', style)
+    return pjoin(folder, fname)
+
+
+def read_bundles_2_subjects(subj_id='subj_1', metrics=['fa'],
+                            bundles=['af.left', 'cst.right', 'cc_1']):
+    r""" Read images and streamlines from 2 subjects of the SNAIL dataset
+
+    Parameters
+    ----------
+    subj_id : string
+        Either ``subj_1`` or ``subj_2``.
+    metrics : list
+        Either ['fa'] or ['t1'] or ['fa', 't1']
+    bundles : list
+        Example ['af.left', 'cst.right', 'cc_1']. See all the available bundles
+        in the ``exp_bundles_maps/bundles_2_subjects`` directory of your
+        ``$HOME/.dipy`` folder.
+
+    Returns
+    -------
+    dix : dict
+        Dictionary with data of the metrics and the bundles as keys.
+
+    Notes
+    -----
+    If you are using these datasets please cite the following publications.
+
+    References
+    ----------
+    .. [1] Renaud, E. M. Descoteaux, M. Bernier, E. Garyfallidis,
+    K. Whittingstall, “Morphology of thalamus, LGN and optic radiation do not
```

Comment:
The non-ASCII characters are: `“` and `”`. Need to be replace or use `# -*- coding: utf-8 -*-`
  File "/mnt/data/research/src/dipy/dipy/data/fetcher.py", line 757
SyntaxError: Non-ASCII character '\xe2' in file /mnt/data/research/src/dipy/dipy/data/fetcher.py on line 758, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details